tests/integration_tests_rust.rs

```diff
@@ -375,6 +376,29 @@ fn onchain_spend_receive() {
 		node_b.list_payments_with_filter(|p| matches!(p.kind, PaymentKind::Onchain { .. }));
 	assert_eq!(node_b_payments.len(), 3);
 
+	let payment_id = PaymentId(txid.to_byte_array());
+	let payment_a = node_a.payment(&payment_id).unwrap();
+	assert_eq!(payment_a.amount_msat, Some(amount_to_send_sats * 1000));
+	assert!(payment_a.fee_paid_msat > Some(0));
```

Comment:
Could you assert that this is `None` when the payment is pending? Maybe add fee checks for lightning payments, too.

Reply:
Oh, right. Should we check it earlier?